Update sql-database-recovery-using-backups.md

allenwux opened the pull request:
Add instruction to restore a deleted database to specified point in time using PowerShell

azurecla commented:
Hi __@ChrisCompy__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <span>
        In order for us to evaluate and accept your PR, we ask that you sign a contribution license agreement. It's all electronic and will take just minutes. I promise there's no faxing. https://cla.azure.com.
    </span>

TTYL, AZPRBOT;

ChrisCompy commented:
I should have checked into azure-content-pr.  killing this pull request and will open another as appropriate in the right branch

allenwux committed:
Update sql-database-recovery-using-backups.md

Add instruction to restore a deleted database to specified point in time using PowerShell
